src/logging.c

```diff
@@ -17,10 +17,15 @@ extern "C"
 {
 #endif
 
+#include <string.h>
+
 #include "rcutils/allocator.h"
+#include "rcutils/get_env.h"
 #include "rcutils/logging.h"
 
 bool g_rcutils_logging_initialized = false;
+const char * g_rcutils_logging_output_format_string = \
+  "[{severity}] [{name}]: {message} {line_number}";
```

Comment:
original comment still applies, I think that should be called `location` rather than `line_number` if kept